addons/mail/static/src/models/rtc_session/rtc_session.js

```diff
@@ -139,34 +139,6 @@ function factory(dependencies) {
             });
         }
 
-        /**
-         * Toggles the deaf state of the current session, this must be a session
-         * of the current partner.
-         */
-        async toggleDeaf() {
```

Comment:
ideally keep the method in stable and remove in master (ok to use the new methods inside it)